Solutions/Menes.Specs/Steps/TestClasses/OperationInvokerTestContext.cs

```diff
@@ -11,31 +11,28 @@ namespace Menes.Specs.Steps.TestClasses
     using Idg.AsyncTest;
 
     using Menes.Auditing;
-    using Menes.Internal;
+    using Menes.Internal;    using Microsoft.Extensions.DependencyInjection;
```

Comment:
In github this looks like these two directives are on the same line. If so, that probably wasn't deliberate?